Physics/Collisions/SimulationManager.cs

```diff
@@ -79,13 +90,49 @@ public SimulationManager(TPoseIntegratorCallbacks poseIntegratorCallbacks, Solve
 
         BufferPool.Take(1, out RayCastingResults);
         _hitHandler = new HitHandler { Hits = RayCastingResults };
+        _timeStepSeconds = timeStepSeconds;
+    }
+
+    /// <summary>
+    /// Starts the simulation's internal clock.
+    /// </summary>
+    public void Start()
```

Comment:
That's intentional, the simulation now starts after all controllers are up, this way the simulation doesn't advance when we're not displaying anything yet (otherwise the car may move a fair bit already), or the bots may start shooting at the player already